### ISSUE TITLE
Disabling for a particular file

### ISSUE THREAD

**Issue by DoctorRyner** [ndmitchell/record-dot-preprocessor#10]
Is it possible to exclude the influence of this ghc-option for a particular file? It conflicts with x filed in my record
```
In the expression:
        Z.lens (\ x -> x (x :: (V3))) (\ c x -> c {x = x} :: (V3))
```

**Comment by ndmitchell** [comment]
I'd prefer to fix the problem if at all possible. Can you explain how it conflicts?

**Comment by DoctorRyner** [comment]
I want to have
```
data V3 = V3
    { x :: Double
    , y :: Double
    , z :: Double
    } deriving (Show)
```
But I can't use x for a field name because of this conflict with lens library (probably)
```
/Users/local/tmp/glossa/src/V3.hs:31:121-173: error:
    • Couldn't match type ‘V3’ with ‘V3 -> Double’
      Expected type: (t1 -> f t2) -> V3 -> f t3
        Actual type: (t1 -> f Double) -> (V3 -> t1) -> f V3
    • In the expression:
        Z.lens (\ x -> x (x :: (V3))) (\ c x -> c {x = x} :: (V3))
      In an equation for ‘Z.fromLabel’:
          Z.fromLabel
            = Z.lens (\ x -> x (x :: (V3))) (\ c x -> c {x = x} :: (V3))
      In the instance declaration for
        ‘Z.IsLabel "x" ((t1 -> f t2) -> V3 -> f t3)’
/Users/local/tmp/glossa/src/V3.hs:31:138: error:
    • Couldn't match type ‘V3 -> Double’ with ‘V3’
      Expected type: V3
        Actual type: V3 -> t1
```

**Comment by ndmitchell** [comment]
Ah, that's easy to fix. I'll make a release, tonight with luck

**Comment by DoctorRyner** [comment]
Thanks, I am looking forward to it

**Comment by ndmitchell** [comment]
Fixed. Available in 0.1.5.
